themes/robbyrussell/robbyrussell.theme.sh

```diff
@@ -0,0 +1,29 @@
+#! bash oh-my-bash.module
+
+# This theme attempts to replicate the default "robbyrussell" theme from ohmyzsh:
+#  https://github.com/ohmyzsh/ohmyzsh/blob/master/themes/robbyrussell.zsh-theme
+
+function _omb_theme_PROMPT_COMMAND() {
+    if [[ "$?" == 0 ]]; then
+        local arrow_color="${_omb_prompt_bold_green}"
+    else
+        local arrow_color="${_omb_prompt_bold_brown}"
+    fi
+
+    local base_directory="${_omb_prompt_bold_teal}\W${_omb_prompt_reset_color}"
+    local SCM_THEME_PROMPT_PREFIX="${_omb_prompt_bold_navy}git:(${_omb_prompt_bold_brown}"
```

Comment:
> Also, I realized that when inside an svn repository, the repo information is still decorated with "git:(...)". If there is a way to check whether we are in a git vs svn repository, I could change this to say "svn" when appropriate.

Can you test this?

```suggestion
    local GIT_THEME_PROMPT_PREFIX="${_omb_prompt_bold_navy}git:(${_omb_prompt_bold_brown}"
    local SVN_THEME_PROMPT_PREFIX="${_omb_prompt_bold_navy}svn:(${_omb_prompt_bold_brown}"
    local HG_THEME_PROMPT_PREFIX="${_omb_prompt_bold_navy}hg:(${_omb_prompt_bold_brown}"
```

Reply:
Yes, that produces the desirable behavior, thanks!

Reply:
Thanks for checking!

Reply:
I've also applied the same change to the `rr` theme and squashed it into your new commit. I have also edited the commit message.